sklearn/utils/__init__.py

```diff
@@ -60,6 +60,26 @@ def safe_mask(X, mask):
     return mask
 
 
+def axis0_safe_slice(X, mask, len_mask):
```

Comment:
RANSAC raises an error if all the data points are classified as outliers. But in this case we don't have a fixed threshold, the term `M*sigma` varies for each iteration